resources/assets/sass/app.scss

```diff
@@ -155,6 +155,24 @@ i.fa {
   background-color: #fff;
 }
 
+/** Make table contents wrap on mobile devices (override bootstrap) **/
+.table-responsive{
+  @media screen and (max-width: $screen-xs-max) {
+    > .table{
+      > thead,
+      > tbody,
+      > tfoot {
+        > tr {
+          > th,
+          > td {
+            white-space: normal;
+          }
+        }
+      }
+    }
+  }
+}
+
```

Comment:
Does something like `.overflowing-cell { white-space:normal; }` not work?

Reply:
override statement is now more specific